styleguide/components/21-atoms/inputs/input-date/input-date.twig

```diff
@@ -1,6 +1,9 @@
-<input type="text"
+<input type="date"
        name="{{ date_name }}"
        id="{{ date_id }}"
-       class="datepicker {% if modifier %}{{ modifier }}{% endif %}"
+       class="{% if modifier %}{{ modifier }}{% endif %}"
        placeholder="dd/mm/yyyy"
```

Comment:
Add a pattern check to match your placeholder for browsers that fall back on input type text.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date#Handling_browser_support